src/Network/Network/PublicIpAddress/NewAzurePublicIpTagCommand.cs

```diff
@@ -37,7 +37,11 @@ public class NewAzurePublicIpTagCommand : NetworkBaseCmdlet
         [ValidateSet(
```

Comment:
@rileymckenna we can remove this `ValidateSet` if you're using the `PSArgumentCompleter` since they accomplish the same thing. The breaking change tool will flag this as an exception, but we'll suppress the exception.